pkg/apis/infinispan/v1/infinispan_types.go

```diff
@@ -214,6 +214,7 @@ const (
 	ConditionStopping           ConditionType = "Stopping"
 	ConditionUpgrade            ConditionType = "Upgrade"
 	ConditionWellFormed         ConditionType = "WellFormed"
+	ConditionCrossSiteFormed    ConditionType = "CrossSiteFormed"
```

Comment:
@dmvolod Are hyphens allowed or are there character restrictions?

@pruivo What do you think about being consistent with bare metal? While we're at this I'm tempted to propose that we align cluster view messages a little. 

What about:

"WellFormed" -> "ClusterViewFormed" 
"CrossSiteFormed" -> "CrossSiteViewFormed"

If that sounds acceptable I can send a PR to Infinispan and align the messages across platforms.

Reply:
It would be nice, but I'm not sure that we can change `WellFormed` -> `ConditionWellFormed` now as this cause a migration problems or we should support both `ClusterViewFormed` and `WellFormed` statuses for well formed state.
@rigazilla @ryanemerson what do you think about it?

Reply:
This might not be the PR to start tackling the conditions but I think we should make sure they are meaningful in isolation and written from a more DevOPs perspective.

@Crumby had a question before about `PreliminaryChecksPassed` because it seemed a little mysterious. IIRC the operator provided an error message and detail on how to resolve it. But the condition itself seemed incomplete or vague. 

For this PR at least I think `CrossSiteViewFormed` is a little more descriptive than just `CrossSiteFormed`. It follows the same language we use in docs "clusters can form a cross-site view" and is clear that we're talking about the view.

Also, maybe nevermind about the hyphens. Let's stick with camel case.

Reply:
> What do you think about being consistent with bare metal? 

@oraNod what do you mean?

Reply:
> @Crumby had a question before about `PreliminaryChecksPassed` because it seemed a little mysterious. IIRC the operator provided an error message and detail on how to resolve it. But the condition itself seemed incomplete or vague.

This issue should be submitted separately.

Reply:
I'm not sure we can change `ConditionWellFormed` value in current time and changes, it's better to align types, but need to rewrite a number of code

Reply:
> > What do you think about being consistent with bare metal?
> 
> @oraNod what do you mean?

@pruivo I mean on bare metal you get `ISPN000439: Received new x-site view:` in logs. My thought here was to have the same, or at least similar, message with the Operator.

Reply:
> It would be nice, but I'm not sure that we can change `WellFormed` -> `ClusterViewFormed` now as this cause a migration problems or we should support both `ClusterViewFormed` and `WellFormed` statuses for well formed state.
> @rigazilla @ryanemerson what do you think about it?

I would keep both for backward comp.
I also agree with @oraNod that conditions should be designed with a devOp pov, maybe in a separated PR

Reply:
@dmvolod maybe let's hold off on changing `WellFormed` or adding an alternative in this PR. I think it's getting into scope creep. I've created this issue to review status conditions from a DevOPs POV: https://github.com/infinispan/infinispan-operator/issues/887 

For this PR, I think `CrossSiteViewFormed` is the way to go.

Reply:
Yes, sure @oraNod will fix